### PR TITLE
refactor: remove substratetestnet

### PR DESCRIPTION
**Pull request by felixfaisal**
I think this PR is good for review, It's a simple PR that removes the `SubstrateTestnet` and based on @Verin1005 said, I don't have to generate types manually as it is done in the CI.

### CODE REVIEW

**Comment by 0xverin** [comment]
> Generate client types, Need help

Did yesterday's discussion help you?

**Comment by felixfaisal** [comment]
> > Generate client types, Need help
> 
> Did yesterday's discussion help you?

Yeah I think I got it, It is automatically generated in CI, So I don't think I have to do anything in that area

File: tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts [hunk 1]
```diff
@@ -41,7 +41,6 @@ export default {
                 "Litmus",
                 "LitentryRococo",
                 "Khala",
-                "SubstrateTestnet",
```
Comment (kziemianek) [inline tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts:44]: You need to fill the gap with `__Unused6: "Null"`

Comment (felixfaisal) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts:44]: Thank @kziemianek! Updated ✅

**Review by 0xverin — APPROVED** [review]
LGTM.Thanks!

**Review by kziemianek — APPROVED** [review]

File: tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default {
...
+                __Unused6: "Null",
```
Comment (Kailai-Wang) [inline tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: The syntax is wrong tho 😂 

Btw is `Web3Network` actually used? Shouldn't we use `CorePrimitivesNetworkWeb3Network` ubiquitously? @Verin1005 I thought this problem was fixed already

Comment (0xverin) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: I checked, and we are still using the `Web3Network` at the moment.

Comment (Kailai-Wang) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: Hmm why is that? I thought it had been mentioned multiple times that we'd need to use auto-generated types :(

Comment (0xverin) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: We can indeed replace the introduced [type](https://github.com/litentry/litentry-parachain/blob/p-92-remove-substratetestnet-from-web3network/tee-worker/ts-tests/integration-tests/di_evm_identity.test.ts#L28) with `CorePrimitivesNetworkWeb3Network`.

But this PR update will have an impact on these [definitions](https://github.com/litentry/litentry-parachain/blob/p-92-remove-substratetestnet-from-web3network/tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts#L36C105-L36C116) , so `Web3Network` actually used.

Comment (Kailai-Wang) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: Thanks for checking, why can't we use `CorePrimitivesNetworkWeb3Network` in the definitions? Or is there a way to do that?

Comment (0xverin) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: I have made updates [here](https://litentry.slack.com/archives/C05328L0GAF/p1703884176997769), and the conclusion is negative. @jonalvarezz 's approach is not feasible in our case; we cannot manually import types every time.

Comment (Kailai-Wang) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: Hmm so we can't define the source like types from registry or lookup?

Comment (0xverin) [reply tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts]: As of now, it seems not possible; it cannot be automatically imported. I've also referenced Moonbeam repo and haven't seen them do something similar at the moment. :(

**Pull request merged**